src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -203,6 +205,38 @@ def manure_spread(aeu):
 
     return n_list, p_list
 
+def ls_factors(lu_strms, areas, avg_slope):
+    results = [0.0] * len(lu_strms)
+    for i in range(len(lu_strms)):
+        results[i] = ls_factor(lu_strms[i] * KM_PER_M, areas[i], avg_slope)
+    return results
+
+def ls_factor(stream_length, area, avg_slope):
+    # units --> stream_length:km, area:hectare, avg_slope:%
+    slope_cell_size = 30 #for posterity
+    if 0 <= avg_slope <= 1.0:
+        m = 0.2
```

Comment:
Since the value of `m` depends on `avg_slope` and nothing else, consider putting this in `ls_factors` so that it is calculated only once, and not for each land use type in the loop